include/core_sys.h

```diff
@@ -5,6 +5,13 @@
 #include "struct_decls/struct_0201CD88_decl.h"
 #include "overlay023/funcptr_ov23_022537D4.h"
 
+typedef enum ButtonMode {
+    BUTTON_MODE_NORMAL = 0,
+    BUTTON_MODE_START_IS_X = 1,
+    BUTTON_MODE_SWAP_XY = 2,
+    BUTTON_MODE_L_IS_A = 3,
+} ButtonMode;
```

Comment:
The [style-guide for gen4](https://docs.google.com/document/d/17D62q1v4dcBmet8T8uYoTAP_0IdGTci0-3p0fPcf-a8/edit) is that we do not typedef enums.

```suggestion
enum ButtonMode {
    BUTTON_MODE_NORMAL = 0,
    BUTTON_MODE_START_IS_X = 1,
    BUTTON_MODE_SWAP_XY = 2,
    BUTTON_MODE_L_IS_A = 3,
};
```